packages/commonwealth/client/scripts/controllers/app/webWallets/walletconnect_web_wallet.ts

```diff
@@ -108,7 +109,7 @@ class WalletConnectWebWalletController implements IWebWallet<string> {
     this._provider = await EthereumProvider.init({
       projectId: '927f4643b1e10ad3dbdbdbdaf9c5fbbe',
       chains: [chainId],
-      methods: ['eth_sendTransaction', 'personal_sign', 'eth_signTypedData_v4'],
+      optionalMethods: ['eth_getBlockByNumber', 'eth_sendTransaction'],
```

Comment:
Any reason we're not using signTypedData?

Reply:
we don't use that anymore as SIWE preferred method is personal_sign. A full code search will show that we dont make any rpc requests with it to confirm.

Reply:
https://docs.metamask.io/wallet/concepts/signing-methods/#personal_sign